scripts/test.sh

```diff
@@ -5,10 +5,5 @@ cp -r fixtures/2.5 fixtures/2.6
 for file in `ls fixtures/*_expected.rb` `ls fixtures/$(ruby -v | grep -o '\d\.\d')/*_expected.rb`
 do
     time ruby --disable=gems src/rubyfmt.rb `echo $file | sed s/expected/actual/` > /tmp/out.rb
-    diff /tmp/out.rb $file
-    if [[ $? -ne 0 ]]
-    then
-        echo "got diff"
-        exit 1
-    fi
+    git diff --no-index /tmp/out.rb $file
```

Comment:
I think `diff -u` (or `diff --unified`) will give the same output format as `git diff --no-index`, and seems more clear to me because we wouldn't be invoking Git to diff two untracked files.

Reply:
I thought since we were in the confounds of a git repository, `git diff --no-index` made sense. But I can see your point about clarity and can agree.

Changed to `diff -u` in 6f2d4d8